scripts/on-prem-archive.sh

```diff
@@ -220,6 +228,10 @@ read_base_plans() {
   done < base-plans.txt
 }
 
+read_hab_plans() {
+    hab_plans=( "hab-sup" "hab-launcher")
```

Comment:
Will this also cover `hab-backline` as an automatic dependency?

Reply:
It will for a full sync. This stanza is only used  with the base-plans flag.